portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,18 +15,33 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Eriyeza!</h1>");
+    
+    //Create a list of comments
+    ArrayList<String> comments = new ArrayList<String>(3);
+    comments.add("What other anime have you watched?");
+    comments.add("How old are your nephews?");
+    comments.add("How do you like U of T?");
+
+    // Convert comments list to JSON
+    Gson gson = new Gson();
+    String json = gson.toJson(comments);
+
+    // Send json to server
+    response.setContentType("applications/json;");
```

Comment:
nit: is this required?

Reply:
I am not too sure if it is required but that is how it was written in the example here: https://github.com/eriyezab/my-portfolio/blob/master/walkthroughs/week-3-server/examples/server-stats/src/main/java/com/google/sps/servlets/ServerStatsServlet.java